_posts/2021-01-01-IMT2020004.markdown

```diff
@@ -0,0 +1,13 @@
+---
+title:  "Sooraj Sathish"
+subtitle: ""
+author: "Sooraj"
+image: "img/IMT2020004-background.jpg"
+date:   2021-01-01 12:12:12
+---
+
+## About me
+
+"Hey I am Sooraj from Bangalore. I love watching anime and listening to podcasts while cycling. Hope the campus opens soon!!" 

```

Comment:
You don't have to put this in quotes :)

Reply:
Do i change it and push it again?

Reply:
Just make a commit on your forked repo with the change